test/integration/generator/content/schedule/part3/index.test.js

```diff
@@ -0,0 +1,67 @@
+jest.mock('../../../../../../app/generator/content/schedule/part3/get-table')
+const { getTable } = require('../../../../../../app/generator/content/schedule/part3/get-table')
+
+const part3 = require('../../../../../../app/generator/content/schedule/part3')
+const { topUpSchedule, reductionSchedule, reductionZeroSchedule } = require('../../../../../mocks/mock-schedule')
+
+describe('schedule part 3 integration with getSummary components', () => {
+  afterEach(() => {
+    jest.clearAllMocks()
+  })
+
+  test('Ensure getTable is called once', () => {
+    part3(topUpSchedule)
+    expect(getTable).toHaveBeenCalledTimes(1)
+  })
+
+  describe('top-up schedule part 3', () => {
+    test('Ensure top-up summary header style is set to header3', () => {
+      const result = part3(topUpSchedule)
+      expect(result.stack[0].style).toBe('header3')
+    })
+
+    test('Ensure top-up summary title text is "Payment schedule"', () => {
+      const result = part3(topUpSchedule)
+      expect(result.stack[0].text).toBe('Payment schedule')
+    })
+
+    test('includes unbreakable instruction when top up schedule', () => {
```

Comment:
Do not need, "when top up schedule" in test name because the describe already tells us this is part of the top-up schedule tests

Reply:
Subsequent for other tests under describes